core/src/main/java/com/predic8/membrane/core/http/Header.java

```diff
@@ -191,6 +191,11 @@ public HeaderField[] getAllHeaderFields() {
 		return fields.toArray(new HeaderField[0]);
 	}
 
+	public boolean contains(String header) {
+		return fields.stream()
+				.anyMatch(headerField -> headerField.getHeaderName().equals(header));
```

Comment:
Do we need ignorecase?
Add tests to HeaderTest

Reply:
Header -> HeaderName class already handles equals ignoring case in it's equals override